src/mscorlib/src/System/Diagnostics/Stackframe.cs

```diff
@@ -12,244 +12,27 @@
 namespace System.Diagnostics
```

Comment:
Will do as part of the integration to CoreCLR - as discussed elsewhere, turns out I need to make the initial check-in to ProjectN.